### DISCUSSION TITLE
[Device Support request] MOES Blind Cover motor TS601 Battery Level

### DISCUSSION THREAD

**Discussion by 1979BAZ** [zigpy/zha-device-handlers#3540] [General]
I have this device, with signature pasted below.
A sensor for battery level woul be great.  Can anyone help with this?

{
  "node_descriptor": {
    "logical_type": 2,
    "complex_descriptor_available": 0,
    "user_descriptor_available": 0,
    "reserved": 0,
    "aps_flags": 0,
    "frequency_band": 8,
    "mac_capability_flags": 128,
    "manufacturer_code": 4098,
    "maximum_buffer_size": 82,
    "maximum_incoming_transfer_size": 82,
    "server_mask": 11264,
    "maximum_outgoing_transfer_size": 82,
    "descriptor_capability_field": 0
  },
  "endpoints": {
    "1": {
      "profile_id": "0x0104",
      "device_type": "0x0202",
      "input_clusters": [
        "0x0000",
        "0x0004",
        "0x0005",
        "0x0102",
        "0xef00"
      ],
      "output_clusters": [
        "0x000a",
        "0x0019"
      ]
    }
  },
  "manufacturer": "_TZE200_icka1clh",
  "model": "TS0601",
  "class": "zhaquirks.tuya.ts0601_cover.TuyaMoesCover0601"
}

**Comment by amitkeret** [comment]
@1979BAZ has this been resolved?
I have the same product witht he same signature, and no battery level sensor.
